reference/dic_tags.rst

```diff
@@ -1422,8 +1423,8 @@ the new loader and tag it with ``twig.loader``:
 
 .. note::
 
-    The ``priority`` value is optional and defaults to ``0``.
-    The higher priority loaders are tried first.
+    The ``priority`` is optional and its value is a positive or negative integer
+    that defaults to ``0``. Higher numbers mean that loaders are tried first.
```

Comment:
"Higher numbers mean that loaders are tried first" reads a little awkwardly to me. "Loaders with higher numbers are tried first." is simpler imo.

Reply:
Thanks for reviewing this. I made the change you proposed. Thanks!